build_tools/circle/build_doc.sh

```diff
@@ -148,8 +148,6 @@ else
     make_args=html
 fi
 
-make_args="SPHINXOPTS=-T $make_args"  # show full traceback on exception
```

Comment:
This is overwriting the `SPHINXOPTS` from the make file.
I prefer to add it directly in the `Makefile`.